pulser/simulation/simulation.py

```diff
@@ -317,14 +318,15 @@ def evaluation_times(self) -> np.ndarray:
     @evaluation_times.setter
     def evaluation_times(self, value: Union[str, ArrayLike, float]) -> None:
         """Sets times at which the results of this simulation are returned."""
+        final_seq_time = (self._tot_duration - 1) / 1000
         if isinstance(value, str):
             if value == "Full":
-                self._eval_times_array = np.append(
-                    self.sampling_times, self._tot_duration / 1000
-                )
+                self._eval_times_array = np.copy(self.sampling_times)
             elif value == "Minimal":
                 self._eval_times_array = np.array(
-                    [self.sampling_times[0], self._tot_duration / 1000]
+                    [
+                        self.sampling_times[0],
```

Comment:
I'm sorry @sebgrijalva , I still don't get it. Can you point me to the line where the `final_seq_time` and the `0` are being injected into `_eval_times_array` for this case?